### PR TITLE
Allow specifying a custom startup command

### PR DESCRIPTION
**Pull request by shrishrirang**
- Currently, a custom startup script can be supported, but a custom startup command is not supported
- This commit adds support for specifying a custom startup command
- If the specified value is not a file path, we assume it must be a valid command and we run it at the time of startup
- If neither is specified, we attempt to run the default startup file (/home/startup.sh)
- Note that, this commit removes support for the deprecated INIT_SCRIPT app setting.

### CODE REVIEW

**Comment by shrishrirang** [comment]
@jvano

File: shared/init_container.sh [hunk 1]
```diff
@@ -38,28 +38,56 @@ eval $(printenv | sed -n "s/^\([^=]\+\)=\(.*\)$/export \1=\2/p" | sed 's/"/\\\"/
...
-if [ -n "$APPSETTING_INIT_SCRIPT" ]
+if [ -n "$STARTUP_COMMAND" ]
 then
-    echo Running custom initialization script "$APPSETTING_INIT_SCRIPT"
-    source $APPSETTING_INIT_SCRIPT
-    initScriptExitCode=$?
-    echo Initialization script exited with code $initScriptExitCode
-    exit $initScriptExitCode
+    echo Running STARTUP_COMMAND: "$STARTUP_COMMAND"
+    $STARTUP_COMMAND
```
Comment (jvano) [inline shared/init_container.sh:87]: >$STARTUP_COMMAND [](start = 4, length = 16)

we assume they will run java, so probably we still should inject our properties like $SPRING_BOOT_PROPS and the logger that we have plans to add in the future. #Resolved

Comment (jvano) [reply shared/init_container.sh:87]: after the command exits, we should exit the container too

---
In reply to: [264938371](https://github.com/Azure-App-Service/java/pull/19#discussion_r264938371) [](ancestors = 264938371)

**Review by jvano — CHANGES_REQUESTED** [review]
:clock1:

File: shared/init_container.sh (hunk 1 above)
Comment (shrishrirang) [reply shared/init_container.sh:87]: do you mean - exit if the startup script fails?


---
In reply to: [264938498](https://github.com/Azure-App-Service/java/pull/19#discussion_r264938498) [](ancestors = 264938498,264938371)

Comment (shrishrirang) [reply shared/init_container.sh:87]: done

---
In reply to: [264948918](https://github.com/Azure-App-Service/java/pull/19#discussion_r264948918) [](ancestors = 264948918,264938498,264938371)

Comment (shrishrirang) [reply shared/init_container.sh:87]: @jvano - merging the PR. Feel free to let me know if you have any comments.

**Pull request merged**